src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,58 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const URL = 'https://api.spacexdata.com/v4/rockets';
+const initialState = {
```

Comment:
I would suggest you move the slice into the Rockets folder in features, it is a common best practice to have everything related to a feature in its feature's folder. here is an example: https://javascript.plainenglish.io/redux-toolkit-the-standard-way-to-write-redux-dcfb372202b8, you may find other sources by googling on the subject if needed.

Reply:
@diarisdiakite, thanks for your feedback. Everything fixed. Ready for the review